docs/axes.md

```diff
@@ -87,6 +87,28 @@ function mySIPrefixFormatter(value, index, scale, tickTotal) {
 }
 ```
 
+**Note!** The return value will be wrapped with a `<text>` node if it's a string, `<tspan>`, or `<textPath>`. In all other cases the returned element will replace the `<text>` node and receive two additional props: `containerWidth` and `tickCount`. This way you can e.g. render a `<div>` to truncate long labels:
+
+```javascript
+const MyLabel = props => (
+  <foreignObject>
+    <div
+      xmlns="http://www.w3.org/1999/xhtml"
+      style={{
+        width: props.containerWidth / props.tickCount, overflow: 'hidden',
+        whiteSpace: 'nowrap'
+      }}
+    >
+      {props.children}
+    </div>
+  </foreignObject>
+);
+
+function myFormatter(value) {
+  return <MyLabel>{value}</MyLabel>;
+}
+```
+
```

Comment:
You can include your new showcase example in here! Just use the magic comment like:

```md
<!-- INJECT:"CustomAxesWithLink" -->
```
But with the appropriate chart name